docs/php/workflows.md

```diff
@@ -189,10 +189,10 @@ Implementations of Workflow1 and Workflow2 can registered with the same worker a
 ## Implementing Workflows
 
 A Workflow implementation implements a Workflow interface.
-Each time a new Workflow execution is started, a new instance of the Workflow implementation object is created.
+Each time a new Worklfow Execution is started, a new instance of the Workflow implementation object is created.
 Then, one of the methods (depending on which Workflow Type has been started) annotated with `#[WorkflowMethod]` is invoked.
-As soon as this method returns, the Workflow execution is closed.
-While Workflow execution is open, it can receive calls to signal and query methods.
+As soon as this method returns, the Worklfow Execution is closed.
+While Worklfow Execution is open, it can receive calls to signal and query methods.
```

Comment:
```suggestion
As soon as this method returns, the Workflow Execution is closed.
While Workflow Execution is open, it can receive calls to signal and query methods.
```